documentation/index.rst

```diff
@@ -26,6 +26,16 @@ check:
    content/developmentworkflow
    content/tracingandprofiling
 
+Kw Handbook
+===========
+
+Device and tips for new contributors.
```

Comment:
On the same note as above, we need to "sell" to the reader that all that new contributors need to know is included in this section. In this sense, reformulate the sentence to vehiculate this information.